aws/resource_aws_route53_record_test.go

```diff
@@ -1273,185 +1273,202 @@ resource "aws_route53_record" "default" {
 resource "aws_route53_zone" "main" {
-    name = "notexample.com"
+  name = "notexample.com"
 }
 
 resource "aws_route53_record" "default" {
-	zone_id = "${aws_route53_zone.main.zone_id}"
-	name = "subdomain"
-	type = "A"
-	ttl = "30"
-	records = ["127.0.0.1", "127.0.0.27"]
+  zone_id = aws_route53_zone.main.zone_id
+  name    = "subdomain"
+  type    = "A"
+  ttl     = "30"
+  records = ["127.0.0.1", "127.0.0.27"]
 }
 
 resource "aws_route53_record" "wildcard" {
-    zone_id = "${aws_route53_zone.main.zone_id}"
-    name = "*.notexample.com"
-    type = "A"
-    ttl = "30"
-    records = ["127.0.0.1"]
+  zone_id = aws_route53_zone.main.zone_id
+  name    = "*.notexample.com"
+  type    = "A"
+  ttl     = "30"
+  records = ["127.0.0.1"]
 }
 `
 
 const testAccRoute53WildCardRecordConfigUpdate = `
 resource "aws_route53_zone" "main" {
-    name = "notexample.com"
+  name = "notexample.com"
 }
 
 resource "aws_route53_record" "default" {
-	zone_id = "${aws_route53_zone.main.zone_id}"
-	name = "subdomain"
-	type = "A"
-	ttl = "30"
-	records = ["127.0.0.1", "127.0.0.27"]
+  zone_id = aws_route53_zone.main.zone_id
+  name    = "subdomain"
+  type    = "A"
+  ttl     = "30"
+  records = ["127.0.0.1", "127.0.0.27"]
 }
 
 resource "aws_route53_record" "wildcard" {
-    zone_id = "${aws_route53_zone.main.zone_id}"
-    name = "*.notexample.com"
-    type = "A"
-    ttl = "60"
-    records = ["127.0.0.1"]
+  zone_id = aws_route53_zone.main.zone_id
+  name    = "*.notexample.com"
+  type    = "A"
+  ttl     = "60"
+  records = ["127.0.0.1"]
 }
 `
+
 const testAccRoute53RecordConfigTXT = `
 resource "aws_route53_zone" "main" {
-	name = "notexample.com"
+  name = "notexample.com"
 }
 
 resource "aws_route53_record" "default" {
-	zone_id = "/hostedzone/${aws_route53_zone.main.zone_id}"
-	name = "subdomain"
-	type = "TXT"
-	ttl = "30"
-	records = ["lalalala"]
+  zone_id = "/hostedzone/${aws_route53_zone.main.zone_id}"
+  name    = "subdomain"
+  type    = "TXT"
+  ttl     = "30"
+  records = ["lalalala"]
 }
 `
+
 const testAccRoute53RecordConfigSPF = `
 resource "aws_route53_zone" "main" {
-	name = "notexample.com"
+  name = "notexample.com"
 }
 
 resource "aws_route53_record" "default" {
-	zone_id = "${aws_route53_zone.main.zone_id}"
-	name = "test"
-	type = "SPF"
-	ttl = "30"
-	records = ["include:notexample.com"]
+  zone_id = aws_route53_zone.main.zone_id
+  name    = "test"
+  type    = "SPF"
+  ttl     = "30"
+  records = ["include:notexample.com"]
 }
-
 `
+
 const testAccRoute53RecordConfigCAA = `
 resource "aws_route53_zone" "main" {
-	name = "notexample.com"
+  name = "notexample.com"
 }
 
 resource "aws_route53_record" "default" {
-	zone_id = "${aws_route53_zone.main.zone_id}"
-	name = "test"
-	type = "CAA"
-	ttl = "30"
-	records = ["0 issue \"exampleca.com;\""]
+  zone_id = aws_route53_zone.main.zone_id
+  name    = "test"
+  type    = "CAA"
+  ttl     = "30"
+
+  records = [<<EOF
+0 issue "exampleca.com;"
+EOF
+
+  ]
```

Comment:
This could probably be left as one line

Reply:
Nice catch! I try to undo these bad HEREDOCs but missed a couple.